doc/docs.md

```diff
@@ -611,7 +611,7 @@ will have the type of `f64`.
 
 ### Arrays
 #### Basic Array Concepts
-Arrays are collections of data elements of the same type. They can be represented by
+Arrays are **collections of data elements** of the **same type**. They can be represented by
```

Comment:
Hmm... bold face outside headers) should really be reserved for exceptional warnings. Arrays like these exist in many other programming languages and are really not exceptional.